py/objgenerator.c

```diff
@@ -46,25 +45,6 @@ static
 mp_obj_exception_t mp_static_GeneratorExit_obj;
 #endif
 
-/******************************************************************************/
```

Comment:
I move this struct to the header file so that it can be used in other modules, namely for the implementation of `iscoroutine`.